src/guides/v2.4/config-guide/cli/config-cli-subcommands-static-view.md

```diff
@@ -252,6 +252,15 @@ The following table explains this command's parameters and values.
             <p>No</p>
         </td>
     </tr>
+    <tr>
+        <td>--no-parent</td>
+        <td>
+            <p>This option will be available from 2.4.2 version.It provides ability to deploy the only specific theme(s) for multiple locales, so it will improve build phase of deployment for the those who use single theme with different locales.</p>
```

Comment:
Good job! 

Only one thing: since 2.4.2 has been released, I would say "This option is available from 2.4.2"